Example/ViewController.swift

```diff
@@ -17,12 +17,25 @@ class ViewController: UIViewController {
     
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
+
+        let coordinates = [CLLocationCoordinate2D(latitude: 45.4562, longitude: -122.8793),
+                           CLLocationCoordinate2D(latitude: 45.4562, longitude: -122.4645),
+                           CLLocationCoordinate2D(latitude: 45.6582, longitude: -122.4645),
+                           CLLocationCoordinate2D(latitude: 45.6582, longitude: -122.8793),
+                           CLLocationCoordinate2D(latitude: 45.4562, longitude: -122.8793)]
+        
+        let path = Path(coordinates: coordinates)
+        path.fillColor = UIColor.red.withAlphaComponent(0.5)
+        path.strokeColor = UIColor.green.withAlphaComponent(0.5)
```

Comment:
Should be able to drop UIColor here but apparently, a type cannot be inferred with a subsequent method call.